frontend/i18next-parser.config.js

```diff
@@ -0,0 +1,88 @@
+const SUPPORTED_LOCALES = {
+  en: 'English',
+  zh: '中文',
+  ja: '日本語',
+};
+
+const FALLBACK_LOCALE = 'en';
+
+/*eslint no-undef: "error"*/
+/*eslint-env node*/
+
+module.exports = {
+  SUPPORTED_LOCALES,
+
+  FALLBACK_LOCALE,
+
+  contextSeparator: '_',
+  // Key separator used in your translation keys
+
+  createOldCatalogs: true,
+  // Save the \_old files
+
+  defaultNamespace: 'public',
+  // Default namespace used in your i18next config
+
+  defaultValue: '',
+  // Default value to give to empty keys
+
+  indentation: 2,
+  // Indentation of the catalog files
+
+  keepRemoved: false,
+  // Keep keys from the catalog that are no longer in code
+
+  keySeparator: false,
+  // Key separator used in your translation keys
+  // If you want to use plain english keys, separators such as `.` and `:` will conflict. You might want to set `keySeparator: false` and `namespaceSeparator: false`. That way, `t('Status: Loading...')` will not think that there are a namespace and three separator dots for instance.
```

Comment:
s/plan english/Plain English

    `namespaceSeparator: false` so that, for example, `t('Status: Loading...')` will not think that there is a namespace and three separator dots.